python/samples/kernel-syntax-examples/azure_chat_gpt_with_data_api.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import asyncio
+
+import semantic_kernel as sk
+import semantic_kernel.connectors.ai.open_ai as sk_oai
+
+kernel = sk.Kernel()
+
+# Load Azure OpenAI Settings
+deployment, api_key, endpoint = sk.azure_openai_settings_from_dot_env()
+
+# Load Azure OpenAI with data settings
+azure_aisearch_datasource = sk.azure_aisearch_settings_from_dot_env_as_datasource()
+
+azure_chat_with_data_settings = (
+    sk.PromptTemplateWithDataConfig.AzureChatWithDataSettings(
+        dataSources=[azure_aisearch_datasource]
```

Comment:
I would really like to see the field mapping in here (or a seperate example)

Reply:
Good idea - added field mapping to this example to demonstrate it